vault/dendron.topic.publish.config.enableMermaid.md

```diff
@@ -0,0 +1,21 @@
+---
+id: wT7zkio7u4hnQc1vk5Dpg
+title: enableMermaid
+desc: 'Enable rendering [[Mermaid diagrams|dendron.topic.markdown#diagrams]] in published notes.'
```

Comment:
i don't think so but @SeriousBug can confirm

Reply:
@ScriptAutomate Rename will fix up tags in the frontmatter (only ones under `tags: ...`), but that's all as far as I know.

Reply:
Just tested and it looks like it doesn't. If we do a rename (F2) from within the `desc` it will, but if the source was refactored this won't get updated.

I'll change all wikilinks that are in frontmatter to be plain text and add the wikilink in the body.

Good catch!

Reply:
removed all wikilinks and markdown links from frontmatter.

we can later enhance the update behavior to also update the wikilinks in frontmatter, but I figured having wikilink / markdown syntax in description would clog up the lookup quickpick label too much anyway